prime-router/metadata/fhir_mapping/hl7/segments/ORC/DiagnosticReport.yml

```diff
@@ -0,0 +1,324 @@
+      expressionType: JEXL
+    _end:
+      expressionType: nested
+      condition: $endDate NOT_NULL
+      expressionsMap:
+        extension_1:
+          generateList: true
+          expressionType: nested
+          expressionsMap:
+            url:
+              type: SYSTEM_URL
+              value: hl7v2-date-time
+            valueString:
+              type: STRING
+              valueOf: $endDate
+              expressionType: HL7Spec
+
+issued:
+  type: STRING
+  valueOf: 'GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)'
+  expressionType: JEXL
+  vars:
+    dateTimeIn: OBR.22
+_issued:
+  expressionType: nested
+  condition: $obr22 NOT_NULL
+  vars:
+    obr22: OBR.22
+  expressionsMap:
+    extension_1:
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: hl7v2-date-time
+        valueString:
+          type: STRING
+          valueOf: $obr22
+          expressionType: HL7Spec
+
+category:
+  expressionType: resource
+  valueOf: datatypes/ID/CodeableConcept
+  specs: OBR.24
+
+status:
+  type: DIAGNOSTIC_REPORT_STATUS
+  valueOf: OBR.25
+  expressionType: HL7Spec
+  required: true
+
+resultsInterpreter:
+  vars:
+    obr32: STRING_ALL, OBR.32
+  condition: $obr32 NOT_NULL
+  valueOf: datatypes/NDL/PractitionerRole
+  generateList: true
+  expressionType: reference
+  specs: OBR.32
+
+performer_1-technician:
+  valueOf: datatypes/NDL/PractitionerRole
+  condition: $obr34 NOT_NULL
+  generateList: true
+  expressionType: reference
+  specs: OBR.34
+  vars:
+    obr34: OBR.34
+  constants:
+    performerParticipationType: "SPRF"
+
+performer_2-transcriptionist:
+  valueOf: datatypes/NDL/PractitionerRole
+  condition: $obr35 NOT_NULL
+  generateList: true
+  expressionType: reference
+  specs: OBR.35
+  vars:
+    obr35: OBR.35
+  constants:
+    performerParticipationType: "TRANS"
+
+extension:
+  generateList: true
+  expressionType: nested
+  expressions:
+    - expressionType: nested
+      vars:
+        orc16: ORC.16
+      condition: $orc16 NOT_NULL
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: "http://hl7.org/fhir/StructureDefinition/event-statusReason"
+        valueCodeableConcept:
+          expressionType: resource
+          valueOf: datatypes/CWE/CodeableConcept
+          specs: ORC.16
+
+encounter:
+  valueOf: datatype/Reference
+  expressionType: resource
+  specs: $Encounter
+
+subject:
+  valueOf: datatype/Reference
+  expressionType: resource
+  specs: $Patient
+
+basedOn:
+  valueOf: datatype/Reference
+  expressionType: resource
+  specs: $ServiceRequest
+  useGroup: true
+
+specimen:
+  valueOf: datatype/Reference
+  generateList: true
+  expressionType: resource
+  specs: $Specimen
+  useGroup: true
+
+result:
+  valueOf: datatype/Reference
+  generateList: true
+  expressionType: resource
+  specs: $Observation
+  useGroup: true
+
+
+#extension_3:
```

Comment:
Remove comment? None of these are in the mapping spreadsheet, were they in NIST?

Reply:
Nope, this is dead stuff, I'll remove it